### ISSUE TITLE
Add amplify icon and support for amplify card property

### ISSUE THREAD

**Issue by jordanweiler** [zzorba/marvelsdb#250]
We'll need to add the amplify icon to the codebase since it doesn't exist yet. We should be able to render the icon in text via `[amplify]`.

We need to also support a new `scheme_amplify` property on card data. It should behave similarly to `scheme_acceleration`, `scheme_crisis`, and `scheme_hazard`. This property should display the amplify icon when rendered.

**Comment by mmougy** [comment]
apparently [marvel-icons.ttf](https://github.com/zzorba/marvelsdb/blob/e4cc2ca74fc5358c800d68112fcfe03a242c8382/src/AppBundle/Resources/public/fonts/marvel-icons.ttf) does not contain the amplify icon<img src="https://github.com/user-attachments/assets/3498f759-8e2e-45fc-950a-a25011c0ce2d" width="30px"/>

![image](https://github.com/user-attachments/assets/76839a69-ed76-4cac-83c7-97ac02cb77da)

does it exist a newer version of [marvel-icons.ttf](https://github.com/zzorba/marvelsdb/blob/e4cc2ca74fc5358c800d68112fcfe03a242c8382/src/AppBundle/Resources/public/fonts/marvel-icons.ttf)?

**Comment by Freakmod** [comment]
i have a other marvel-icons.ttf with amplify 
![Champions Icons Font Guide](https://github.com/user-attachments/assets/fdb7b747-f10b-4774-8719-1521f9f2cb7a)
[ChampionsIcons.zip](https://github.com/user-attachments/files/17815386/ChampionsIcons.zip)
 if any option to put the icon in the web

**Comment by jordanweiler** [comment]
After some thought, I don't think `scheme_amplify` is an appropriate property field name for this new feature. Yes, this would be consistent with the existing `scheme_acceleration`, `scheme_crisis`, and `scheme_hazard` properties that are similar in nature but I believe those property names are also incorrectly named.

The amplify icon doesn't have anything to do with scheme, whether a main/side/player side scheme or the act of scheming. Tying the amplify icon with the term `scheme` would add a needless association with a word that has several meanings in Marvel Champions and doesn't impact when this property is used. The amplify icon has the following text in the 1.6 rules:
> When a boost card is turned faceup during an enemy activation, add one additional boost icon to that card for each amplify icon in play.

I believe the JSON property should just be called `amplify` and it should be a number type. I've updated the title of this issue to reflect this.

Separately from this issue, it might be worth considering whether the existing `scheme_acceleration`, `scheme_crisis`, and `scheme_hazard` should also be renamed to just `acceleration`, `crisis`, and `hazard`. Looking at the core set and the original 1.0 rules, these properties were only ever used in side_schemes. Since that type, they can show up in a variety of card types so having `scheme` as part of the name doesn't make a lot of sense. I'm guessing the name was originally added because only those card types used this property.

**Comment by mmougy** [comment]
I agree "scheme_acceleration" "scheme_crisis" ... are not very representative name

but them having the same prefix is quite handy in files 
if we decide to change this in a future issue, i'll propose something like:

icon_acceleration
icon_amplify
icon_crisis
icon_hazard

**Comment by jordanweiler** [comment]
I'm not sure I agree that having the same prefix is handy in files. Most cards don't set more than one of these four properties and each of these properties means something different. I don't think adding `icon` to the property necessarily helps anything if the property name is clear.

There are other icons that don't have an `icon_` prefix in their property (boost, per_player, star, unique, etc) so there's not a precedence of having an `icon_` prefix for properties and if we tried to create that as a new pattern I think it's just adds unnecessary complexity to already clear property names.
